var/spack/repos/builtin/packages/py-lxml/package.py

```diff
@@ -27,8 +28,9 @@ class PyLxml(PythonPackage):
 
     depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
     depends_on('py-setuptools', type='build')
-    depends_on('libxml2', type=('build', 'link', 'run'))
-    depends_on('libxslt', type=('build', 'link', 'run'))
+    depends_on('libxml2@2.9.2:', type=('build', 'link', 'run'))
+    depends_on('libxslt@1.1.28:', type=('build', 'link', 'run'))
+    depends_on('py-cython@0.29.7:', type=('build', 'run'))
```

Comment:
Is cython actually required to build from the release tarballs or only for git checkouts? A lot of times releases come with files pre-cythonized.

Reply:
It seems that you are right and cython is only required when built from github (pxi and pxd files are included in the tarball). Building and executing also work without it. Sorry, I was not aware that this can be the case.